DCode.Web/Views/ReportingBase/Base.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿
+@{
+    Layout = "~/Views/Shared/_LayoutReporting.cshtml";
+}
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@ViewBag.Title - Reporting Page</title>
+    @Styles.Render("~/Content")
+    <link href="~/Content/bootstrap.css" rel="stylesheet" />
+    @* @Scripts.Render("~/bundles/")*@
+</head>
+<body>
+    <div class="navbar navbar-inverse navbar-fixed-top">
+      Reporting content here   
+    </div>
+
+  
+
+
+    @Scripts.Render("~/bundles/jquery")
```

Comment:
`jquery` & `bootstrap` have already been loaded in `_layoutReporting.cshtml`. Is there a specific need to re-render them?